pkg/blockdevice/probe/probe.go

```diff
@@ -178,6 +180,49 @@ func probe(devpath string) (devpaths []string) {
 	return devpaths
 }
 
+// GetBlockDeviceWithPartitonName probes all known block device's partition
+// table for a parition with the specified name.
+func GetBlockDeviceWithPartitonName(name string) (bd *blockdevice.BlockDevice, err error) {
```

Comment:
This is required to do a lookup up the partitions that we never create a filesystem on.